htdocs/core/lib/pdf.lib.php

```diff
@@ -2500,13 +2500,7 @@ function pdf_getLinkedObjects(&$object, $outputlangs)
 		} elseif ($objecttype == 'commande' || $objecttype == 'supplier_order') {
 			$outputlangs->load('orders');
 
-			if (count($objects) > 1 && count($objects) <= (getDolGlobalInt("MAXREFONDOC") ? getDolGlobalInt("MAXREFONDOC") : 10)) {
```

Comment:
Removing generation from pdf_getLinkedObjects() to replace it by public_note seems a good idea.
However, the option getDolGlobalInt("MAXREFONDOC") must still be used to limit size of the public_note.